tools/create-translation-issues.sh

```diff
@@ -94,7 +96,7 @@ We are getting ready for the ${RELEASE} release and have created the [${TRANSLAT
 
 We would be happy if you updated the translations for **${LANG}** by **${DEADLINE}**.
 
-Please
+Please either [update the translations on Weblate](https://hosted.weblate.org/projects/jamulus/) or use git:
 
 - Start your work in your fork on a branch based on jamuluswebsite'"'"'s `${TRANSLATE_BRANCH}` branch.
 - Update the language-specific files using your favorite editor (or directly on Github),
```

Comment:
```suggestion
- Update the language-specific files using your favourite editor (or directly on GitHub),
```

Reply:
Whereas you're right about "GitHub": should use the styling they provide.

Reply:
Done.